fileio/ft_write_data.m

```diff
@@ -917,3 +1002,61 @@ function ft_write_data(filename, dat, varargin)
       type{i} = 'Other';
   end
 end
+
+function pos = getorthoviewpos(pos, coordsys, viewpoint)
```

Comment:
it would indeed be good to make this a regular private function. A copy is needed in `fieldtrip/private` and another copy in `fieldtrip/fileio/private`, and they should both be in `bin/synchronize-private.sh`. If you make a private function, please also update the "see also" help in `private/setviewpoint.m`